Task-38346: Long breadcrumb UI issue document preview.

montassarhamadi opened the pull request:
Problem: The document  is previewed with bad Ui for Long breadcrumb (the folder names are shown badly).
Fix: Refine the UI for long breadcrumb ( we can add ellipses as an example).

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

montassarhamadi committed:
Task-38346:Long breadcrumb UI issue document preview (#327)

Problem:The document  is previewed with bad Ui for Long breadcrumb (the folder names are shown badly)
Fix:Refine the UI for long breadcrumb

montassarhamadi committed:
Task-38346:Long breadcrumb UI issue document preview (#327)

Problem:The document  is previewed with bad Ui for Long breadcrumb (the folder names are shown badly)
Fix:Refine the UI for long breadcrumb

montassarhamadi committed:
Task-38346: Long breadcrumb UI issue document preview.
Problem: The document  is previewed with bad Ui for Long breadcrumb (the folder names are shown badly).
Fix: Refine the UI for long breadcrumb ( we can add ellipses as an example).

Amine0008 committed:
changed v-dialog selector to v-footer  (#328)

ISSUE: TABS bar of space hide the footer of the drawer

FIX: changed style selector from v-dialog ( previous Vue component) to v-footer (new Vue component), so that the same styles apply to the new component.

Amine0008 committed:
Task5039 bug strange grey bar displayed in multiple places in mobile app (#334)

issue: task #50398 [BUG] Strange grey bar displayed in multiple places in mobile app
fixed: changed the hardcoded zindex for drawer-parent class to a value of 'auto'

montassarhamadi committed:
improve breadCrumb UI in document preview

AzmiTouil commented on platform-ui-skin/src/main/webapp/skin/less/helpers.less:
```diff
@@ -234,10 +234,12 @@
+    font-size: 25px;
     padding-right: 5px;
     margin: auto;
+    color: #578dc9;
```
```suggestion
    color: @primaryColor;
```

abenmbarek committed:
TASK-35353 : In User card the pull-down options is bad display (#333)

ISSUE : In User card the pull-down options is bad display (french language)
FIX : fix the refuse connect button width and add an ellipsis to the text button in "refuseToConnectButton" css class.

montassarhamadi committed:
used variable primaryColor for ellipsis  instead of static color

AzmiTouil committed:
Merge branch 'feature/maintenance' into task-38346

montassarhamadi committed:
Merge branch 'feature/maintenance' into task-38346
